src/util/VehicleSpawner.cpp

```diff
@@ -21,17 +20,25 @@ namespace YimMenu
 
 			Vector3 dir = ENTITY::GET_ENTITY_FORWARD_VECTOR(Self::PlayerPed);
 			float rot   = (ENTITY::GET_ENTITY_ROTATION(Self::PlayerPed, 0)).z;
-
-			Vehicle veh =
-			    VEHICLE::CREATE_VEHICLE(model, pCoords.x + (dir.x * offset), pCoords.y + (dir.y * offset), pCoords.z, rot, 1, 1, 0, 0);
-
+			Vector3 pCoords;
+			Vehicle veh;
+			if (coords)
+			{
+				pCoords = coords.value();
+				veh     = VEHICLE::CREATE_VEHICLE(model, pCoords.x, pCoords.y, pCoords.z, rot, 1, 1, 0, 0);
+			}
+			else
+			{
+				pCoords = ENTITY::GET_OFFSET_FROM_ENTITY_IN_WORLD_COORDS(player_ped_id, 0.0, -10.0, 0.0);
```

Comment:
Why behind?

Reply:
Why not behind 🤣

Reply:
@DayibBaba do you want this to be changed?